src/qibocal/auto/transpile.py

```diff
@@ -97,19 +98,65 @@ def execute_transpiled_circuit(
         backend,
         transpiler,
     )[0]
-
     return transpiled_circ, backend.execute_circuit(
         transpiled_circ, initial_state=initial_state, nshots=nshots
     )
 
 
+def get_natives(platform):
```

Comment:
Style proposal:
```suggestion
def natives(platform):
```
`get_` usually adds nothing...

Reply:
Yes, but `natives` is already used as variable in this file.

Reply:
It is used locally, so there is no clash.

In particular, where it is used as a local variable, `get_natives` is not used, so there is no danger in shadowing.
In any case, if you want to strongly prevent clashes, you could always append a `_` to the local name (i.e. `natives_`), or anything similar.

However, this one was definitely about style. Feel free to ignore it :)